searches/exponentialsearch.py

```diff
@@ -0,0 +1,80 @@
+# Author:- Arnab Nath
+# Python program to find an element x
+# in a sorted array using Exponential Search
+
+# A recursive binary search function returns
+# location of x in given array arr[l..r] is
+# present, otherwise -1
+arr = [4, 7, 10, 34, 44, 70]
+n = len(arr)
+x = 44
+if arr[0] == x:
+    print("0")
+i = 1
+# Finding range for binarySearch
+while i < n and arr[i] <= x:
+    i = i * 2
+min1 = min(i, len(arr))
```

Comment:
Please read CONTRIBUTING.md...  This code needs to go under `if __name__ == "__main__": because we do not want in to run everytime we run pytest.